header-only/layout-builder/awkward/LayoutBuilder.h

```diff
@@ -1129,43 +1126,25 @@ namespace awkward {
       /// @brief Inserts the last valid index in the `index` buffer and
       /// returns the reference to the builder content.
       BUILDER&
-      append_valid() noexcept {
-        last_valid_ = content_.length();
-        index_.append(last_valid_);
+      append() noexcept {
```

Comment:
I think we would want the method name to be different than what we have for `Numpy`. It can be `append_index` maybe.